docs/tutorials/abn/abn.md

```diff
@@ -61,6 +61,7 @@ A sample application using the Iter8 SDK is provided. Deploy both the frontend a
 
 In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A `ConfigMap` is used to describe the make up of possible versions:
 
+<!-- TODO: should this be bumped to v0.16? -->
```

Comment:
Regarding the name "Iter8 A/B/n Service" perhaps use Iter8 service (or controller -- whatever we decide). No reason to have a different name here.